01week/rockPaperScissors.js

```diff
@@ -8,10 +8,65 @@ const rl = readline.createInterface({
 });
 
 
+
+// formatHand takes one string argument and either returns 1 of 3 possble hands for rock paper scissors or undefined
+// Must trim() whitespace and toLowerCase()
+// Use first letter of hand to determine if it should be rock || paper || scissors
+// methods: switch, charAt(), trim(), toLowerCase()
+const formatHand = (hand) => {
+  let setHand;
+  switch(hand.trim().toLowerCase().charAt(0)){
+    case 'r':
+      setHand = 'rock';
+      break;
+    case 'p':
+      setHand = 'paper';
+       break;
+    case 's':
+      setHand = 'scissors';
+      break;
+    default:
+      break;
+  }
+  return setHand;
+}
+
+// Create rockPaperScissors Parent function that takes two string arguments and returns 1 winner
+// Check if arguments are truthy
+// Create new variables to hold formatted arguments
+// Check...
+  // if hand1 is the same as hand2,       return 'It's a tie!'
+  // if hand1 = rock & hand2 = paper,     return 'Hand two wins!'
+  // if hand1 = paper & hand2 = scissors, return 'Hand two wins!'
+  // if hand1 = scissors & hand2 = rock,  return 'Hand two wins!' 
+  // if hand2 = rock & hand1 = paper,     return 'Hand one wins!' 
+  // if hand2 = paper & hand1 = scissors, return 'Hand one wins!'
+// if hand2 = scissors & hand1 = rock,  return 'Hand one wins!' 
+// methods: if statements, 
 function rockPaperScissors(hand1, hand2) {
 
   // Write code here
-
+  
+  if(hand1 && hand2){
+    const handOne = formatHand(hand1);
+    const handTwo = formatHand(hand2);
+    if(handOne && handTwo){
+      if(handOne === handTwo){
+        return 'It\'s a tie!'
+      }
+      if(handOne == 'rock'){
+        return handTwo == 'paper' ? 'Hand two wins!' : 'Hand one wins!';
+      } 
+      if(handOne == 'paper'){
+        return handTwo == 'scissors' ? 'Hand two wins!' : 'Hand one wins!';
+      } 
+      if(handOne == 'scissors'){
```

Comment:
this check is redundant, the only other option handOne could be is scissors. Just do an else on the paper